apps/crn-frontend/package.json

```diff
@@ -76,8 +79,9 @@
     "jose": "2.0.6",
     "nock": "13.0.11",
     "pirates": "4.0.5",
-    "react-scripts": "4.0.3",
+    "react-scripts": "5.0.1",
     "react-test-renderer": "17.0.2",
-    "typescript": "4.6.4"
+    "typescript": "4.6.4",
+    "webpack": "5.74.0"
```

Comment:
Not sure we need this 🤷

Reply:
i dont think we do, no, this was for react-app-rewired